app/Resources/views/programs/index.html.twig

```diff
@@ -19,22 +19,22 @@
             <thead>
                 <tr>
                     <th>{{ msg('program') }}</th>
-                    <th></th>
-                    <th class="center-align-cell">{{ msg('events') }}</th>
-                    <th class="center-align-cell">{{ msg('participants') }}</th>
-                    <th class="center-align-cell">{{ msg('new-editors') }}</th>
-                    <th class="center-align-cell">{{ msg('day-retention', [30]) }}</th>
-                    <th class="center-align-cell">{{ msg('pages-created') }}</th>
-                    <th class="center-align-cell">{{ msg('pages-improved') }}</th>
+                    <th class="text-center text-nowrap"></th>
+                    <th class="text-center text-nowrap">{{ msg('events') }}</th>
+                    <th class="text-center text-nowrap">{{ msg('participants') }}</th>
+                    <th class="text-center text-nowrap">{{ msg('new-editors') }}</th>
+                    <th class="text-center text-nowrap">{{ msg('day-retention', [30]) }}</th>
+                    <th class="text-center text-nowrap">{{ msg('pages-created') }}</th>
+                    <th class="text-center text-nowrap">{{ msg('pages-improved') }}</th>
                 </tr>
             </thead>
             <tbody>
                 {% for program in programs %}
-                    <tr class="program-entry">
+                    <tr class="program-entry text-nowrap">
```

Comment:
Trying remove this `text-nowrap`, since we want the title to wrap, and not all the cells including the title.

Reply:
That worked! Silly me, I totally missed that. I messed up while rebasing this to fix conflicts so now it think the autocompletion changes were introduced in this PR. :/ I fixed the other stuff though.
I added another tiny change to not show an empty table if there are no events in a program and instead show a message saying that.